### PR TITLE
added running QISKIT on IBM

### PR DESCRIPTION
**Pull request by ArneBartenbach**
running the "own" format before adding Braket - unitll now only QASM was supported

the main part of running braket on IBM will follow after transpile manager from other branch is included

### CODE REVIEW

File: qunicorn_core/core/pilotmanager/ibm_pilot.py [hunk 1]
```diff
@@ -41,6 +41,8 @@
 class IBMPilot(Pilot):
     """The IBM Pilot"""
 
+    qc = None
```
Comment (st161630) [inline qunicorn_core/core/pilotmanager/ibm_pilot.py:44]: better Naming to better represent usage - maybe short comment on why its here (its a fringe case)

File: qunicorn_core/core/pilotmanager/ibm_pilot.py [hunk 2]
```diff
@@ -133,13 +135,20 @@ def __get_backend_circuits_and_id_for_qiskit_runtime(self, job_dto):
...
         """Transforms the circuit string into IBM QuantumCircuit objects"""
+        global qc
         circuits: list[QuantumCircuit] = []
         error_results: list[ResultDataclass] = []
 
         # transform each circuit into a QuantumCircuit-Object
         for program in job_dto.deployment.programs:
             try:
-                circuits.append(QuantumCircuit().from_qasm_str(program.quantum_circuit))
+                if program.assembler_language == AssemblerLanguage.QASM:
```
Comment (st161630) [inline qunicorn_core/core/pilotmanager/ibm_pilot.py:145]: Some documentation

File: qunicorn_core/core/pilotmanager/ibm_pilot.py [hunk 3]
```diff
@@ -186,8 +195,10 @@ def transpile(self, provider: IBMProvider, job_dto: JobCoreDto):
         # ############################################################################################
 
         circuits: list[QuantumCircuit] = self.__get_circuits_as_QuantumCircuits(job_dto)
+
```
Comment (st161630) [inline qunicorn_core/core/pilotmanager/ibm_pilot.py:198]: remove new lines

File: qunicorn_core/db/cli.py [hunk 4]
```diff
@@ -85,12 +86,19 @@ def get_quasm_string() -> str:
...
+        quantum_circuit=utils.get_default_qasm_string(2), assembler_language=AssemblerLanguage.QASM
+    )
     qasm3_str: str = "OPENQASM 3; \nqubit[3] q;\nbit[3] c;\nh q[0];\ncnot q[0], q[1];\ncnot q[1], q[2];\nc = measure q;"
-    qasm3_program = QuantumProgramDataclass(quantum_circuit=qasm3_str)
+    qasm3_program = QuantumProgramDataclass(quantum_circuit=qasm3_str, assembler_language=AssemblerLanguage.QASM)
     braket_str: str = "Circuit().h(0).cnot(0, 1)"
-    braket_program = QuantumProgramDataclass(quantum_circuit=braket_str)
+    braket_program = QuantumProgramDataclass(quantum_circuit=braket_str, assembler_language=AssemblerLanguage.BRAKET)
+
+    qiskit_str: str = "global qc; qc = QuantumCircuit(2, 2);qc.h(0);qc.cx(0, 1);qc.measure(0, 0);qc.measure(1, 1)"
```
Comment (st161630) [inline qunicorn_core/db/cli.py:100]: check if possible easier option might be possible

File: qunicorn_core/static/enums/assembler_languages.py [hunk 5]
```diff
@@ -19,11 +19,13 @@ class AssemblerLanguage(StrEnum):
     """Enum to save the different assembler languages
 
     Values:
-        QASM: Standard assembler format from IBM
+        QASM: Standardised assembler format
```
Comment (st161630) [inline qunicorn_core/static/enums/assembler_languages.py:22]: leave standard - good to remove IBM

File: qunicorn_core/db/cli.py (hunk 4 above)
Comment (ArneBartenbach) [reply qunicorn_core/db/cli.py:100]: TODO is added to the ticket

**Review by st161630 — APPROVED** [review]

**Pull request merged**